src/data/showcases.js

```diff
@@ -1127,6 +1127,16 @@ export const Showcases = [
     source: null,
     tags: ["lending", "token", "marketplace", "nftsupport"],
   },
+  {
+    title: "NFTify",
+    description:
+      "The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch and trade their NFT assets without friction, coding skills – at the lowest possible costs.",
```

Comment:
We don't have such promotional language for projects on the Dev Portal so please rewrite this as a straightforward list of features... especially those features that may be _unique_.

Reply:
Got it! I have changed the description.

Reply:
I see it's changed, but not enough in my opinion.  Please tell your boss (re: https://github.com/cardano-foundation/developer-portal/pull/1204#discussion_r1477736962) by reference to this PR thread that we simply cannot list anything which is described with promotional language; currently:
> The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch an NFT store and trade their NFT assets",

Realistically: if you were the "leading" NFT marketplace, you would have been on the Developer Portal already.  I am speaking as another marketer who also recognises the need to keep things *simple and dry* when necessary.  As we generally insist & already said above, just stick to the features please.

Reply:
@rphair I have updated the description. Please check.